common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2111,7 +2111,7 @@
   },
   "INVALID_EMPTY_LOCATION" : {
     "message" : [
-      "The location name cannot be empty string, but `<location>` was given."
+      "The <locationKey> location cannot be empty string or null, but `<locationValue>` was given."
```

Comment:
I thought we agreed to not change this error class...  If we do want the "locationKey", I'd prefer to use sub error conditions, as this is a known set and may need to be translated when supporting international error messages.

Reply:
Do you mean we unparameterised it and given the full set here？

Reply:
Keep INVALID_EMPTY_LOCATION untouched and only add INVALID_LOCATION to align with it